pyglet/font/win32query.py

```diff
@@ -340,7 +340,11 @@ def _enum_font_names(logfont, textmetricex, fonttype, param):
     global FONTDB
 
     lf = logfont.contents
-    name = lf.lfFaceName.decode('utf-8')
+    try:
+        name = lf.lfFaceName.decode('utf-8')
+    except UnicodeDecodeError:
+        name = lf.lfFaceName.decode('ANSI')
```

Comment:
bad windows

Reply:
this is may a way on how to fix #524

Reply:
If possible, can you move these changes to a new pull request?

Reply:
> If possible, can you move these changes to a new pull request?

hmmmm，i'll try